Add To be defined release date with some documentation (#82)
camptocamp/c2cciutils: README.md

-   The `gitattributes` are valid.
-   All text files end with an empty line.

# SECURITY.md

The `SECURITY.md` file should contains the security policy of the repository, espessially the end of
support dates.

> For compatibility with c2cciutils he should contains an array with at least the columns

Reviewer: ```suggestion
For compatibility with c2cciutils it should contain an array with at least the columns
```

> `Version` and `Supported Until`. The `Version` columns will contain the concerned version.

Reviewer: ```suggestion
`Version` and `Supported Until`. The `Version` column will contain the concerned version.
```

The `Supported Until` will contains the date of end of support `dd/mm/yyyy`.

> She also can contains the following sentences:

Reviewer: ```suggestion
It can also contain the following sentences:
```

-   `Unsupported`: no longer supported => no audit, no rebuild.

> -   `Best effort`: the support is ended but it steal rebuilt and audited, but it can stop without any notice.

Reviewer: ```suggestion
-   `Best effort`: the support is ended, it is still rebuilt and audited but this can be can stopped without any notice.
```

> -   `To be defined`: not yet released or the date will be set related of an other project release date (Like for GeoMapFish).

Reviewer: ```suggestion
-   `To be defined`: not yet released or the date will be set related to an other project release date (like for GeoMapFish).
```
